tests/platform_tests/test_platform_info.py

```diff
@@ -155,6 +155,25 @@ def get_psu_num(dut):
 
     return psu_num
 
+def get_healthy_psu_num(duthost):
+    """
+        @Summary: get number of healthy PSUs
+        @param: DUT host instance
+        @return: Number of healthy PSUs
+    """
+    PSUUTIL_CMD = "sudo psuutil status"
+    healthy_psus = 0
+    psuutil_status_output = duthost.command(PSUUTIL_CMD)
+    pdb.set_trace()
+
+    psus_status = psuutil_status_output["stdout_lines"][2:]
+    for iter in psus_status:
+        fields = iter.split()
+        if fields[2] == 'OK':
```

Comment:
Should it be ```fields[-2]``` here? The format of the output is:
```
admin@r-ocelot-07:~$ sudo psuutil status
PSU    Model          Serial          Voltage (V)    Current (A)    Power (W)  Status    LED
-----  -------------  ------------  -------------  -------------  -----------  --------  -----
PSU 1  MTEF-PSF-AC-F  MT2117X17712          12.00          17.75       210.00  OK        green
PSU 2  MTEF-PSF-AC-F  MT2117X17730          12.00          17.75       214.00  OK        green
```
fields[2] is the Model name

Reply:
Hi @congh-nvidia, good catch — we are actually investigating a deeper issue with this and will get back to you asap. Thanks!